source/agora/node/Validator.d

```diff
@@ -222,15 +229,15 @@ public class Validator : FullNode, API
         // Note: Switching the next two lines leads to test failure
         // It should not, and this needs to be fixed eventually
         if (auto timer = this.network.startPeriodicNameRegistration())
-            this.timers ~= timer;
+            this.timers[ValidatorTimers.NameRegistration] = timer;
```

Comment:
Note for later: We might want to unify all those under the same "fire task that rearm themselves" approach. I like it a lot, it's quite elegant.

Reply:
But for the others we do not want to run the task yet.